features/old/accounts/buyers/buyer_password_reset.feature

```diff
@@ -47,9 +47,9 @@ Feature: Buyer password reset
 
   Scenario: Blank passwords
     When I follow "Forgot password?"
-    And I fill in "Email" with "zed@3scale.localhost"
+    And I fill in "Email" with "zed@example.com"
```

Comment:
Is there any technical reason to do this or just better hygiene? I'm fine with the change, just asking.